x-pack/plugins/rollup/public/crud_app/services/breadcrumbs.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CRUD_APP_BASE_PATH } from '../constants';
+
+export const listBreadcrumb = {
```

Comment:
FYI, the breadcrumbs automatically remove the `href` from the last breadcrumb, so you don't have to have two separate versions with/without the href.